packages/hoprd/Dockerfile

```diff
@@ -55,7 +49,7 @@ VOLUME ["/app/db"]
 
 # Admin web server
 EXPOSE 3000
-# REST API 
+# REST API
```

Comment:
Did we open the ports for the `dappnode` compose-like file to include `3001` ?

Reply:
no